front50-sql-postgres/front50-sql-postgres.gradle

```diff
@@ -1,3 +1,3 @@
 dependencies {
-  runtimeOnly "org.postgresql:postgresql:42.2.8"
+  runtimeOnly "org.postgresql:postgresql" // fix CVE-2020-13692 by upgraded version in kork
```

Comment:
I think we don't need the comment here since there aren't any lingering todos, and the commit message has the motivation for the change.

Reply:
Removed comment.
Thanks